### PR TITLE
detail tour

### PR DESCRIPTION
**Pull request by vovanquang12cntt**
Search tours

### CODE REVIEW

**Comment by vovanquang12cntt** [comment]
detail

**Comment by vovanquang12cntt** [comment]
ready

File: app/controllers/tours_controller.rb [hunk 1]
```diff
@@ -1,3 +1,11 @@
 class ToursController < ApplicationController
-  def index; end
+  def show
+    @tour = Tour.find_by id: [params[:id]]
```
Comment (tranducquoc) [inline app/controllers/tours_controller.rb]: remove [] ngoài

**Comment by tranducquoc** [comment]
commented

**Comment by vovanquang12cntt** [comment]
ready

**Comment by vovanquang12cntt** [comment]
hôm nay a review khuya gê hè :)

**Comment by tranducquoc** [comment]
merged

**Pull request merged**